website/pages/blog/guide-aws-org-delegation.mdx

```diff
@@ -0,0 +1,376 @@
+--region us-east-1
+```
+
+| AWS Service | Support via `organizations:RegisterDelegatedAdministrator` |
+| ----------- | ---------- |
+| AWS Trusted Advisor | Yes |
+| AWS CloudFormation StackSets | Yes | 
+| AWS Systems Manager Explorer | Yes | 
+| AWS Systems Manager Change Manager | Yes | 
+| AWS Account Management | Yes | 
+| AWS Backup | Yes | 
+| AWS Compute Optimizer | Yes | 
+| AWS Config | Yes | 
+| AWS DevOps Guru | Yes | 
+| AWS IAM Access Analyzer | Yes | 
+| AWS License Manager | Yes | 
+| AWS Network Manager | Yes | 
+| AWS S3 Storage Lens | Yes | 
+| AWS Service Catalog | Yes | 
+| AWS IAM Identity Center | Yes | 
+| Reachability Analyzer | Yes | 
+
+#### Delegation via Service-specific Actions 
+
+We went through all the current services that support delegated administrator and found the service-specific IAM actions for delegation.  The table below shows the result of our research.
+For these IAM actions, there is a lack of standardization of action naming: for example `EnableOrganizationAdminAccount`, `RegisterOrganizationDelegatedAdmin`, and `AssociateAdminAccount`.
+
+| AWS Service | Service-Specific IAM Action for Delegation |
+| -------- | -------- | 
+| AWS Security Lake (Preview) | securitylake:CreateDatalakeDelegatedAdmin |
+| AWS CloudTrail | cloudtrail:RegisterOrganizationDelegatedAdmin |
+| AWS Firewall Manager | fms:AssociateAdminAccount |
+| AWS GuardDuty | guardduty:EnableOrganizationAdminAccount |
+| AWS Detective | detective:EnableOrganizationAdminAccount |
+| AWS VPC IP Address Manager (EC2 IPAM) | ec2:EnableIpamOrganizationAdminAccount |
+| AWS Security Hub | securityhub:EnableOrganizationAdminAccount |
+| AWS Audit Manager | auditmanager:RegisterOrganizationAdminAccount |
+| AWS Inspector | inspector2:EnableDelegatedAdminAccount |
+| AWS Macie | macie2:EnableOrganizationAdminAccount |
+
+The service specific actions do make child calls to `organizations:RegisterDelegatedAdministrator` and other dependent actions.  The below screenshot shows the associated calls for `fms:AssociateAdminAccount`, `guardduty:EnableOrganizationAdminAccount`, `macie2:EnableOrganizationAdminAccount`, and `inspector2:EnableDelegatedAdminAccount`.
+
+![CloudTrail Event History for Delegated Administrator](/images/blog/guide-aws-org-delegation/cloudtrail-delegating-admin.png)
+
+#### Organization Resource-Based Delegation Policies
+
+Lastly, we have Delegation via Organization Resource-Based Delegation Policies.  This was released in late November of 2022 and allows for more organizational actions outside of the activities listed above. 
+
+This is done via `organizations:PutResourcePolicy`.  Specifically, this enables the following actions that can be grouped into 3 categories: resource tagging, viewing organizational information, and organizational policy management:
+
+##### Organizational Policy Management
+
+Actions:
+* CreatePolicy, AttachPolicy, UpdatePolicy, DeletePolicy, DetachPolicy.
+* DisablePolicyType and EnablePolicyType.
```

Comment:
In most of the other places you use the form `service:action` maybe keep it consistent here?